test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -1933,13 +1933,20 @@ def create_temp_file(self, hostname=None, suffix='', prefix='PtlPbs',
             else:
                 os.write(fd, body)
         os.close(fd)
+
+        if self.get_platform() == 'shasta' and not hostname and asuser:
+            asuser = PbsUser.get_user(asuser)
+            if asuser.host:
+                self.logger.info('set hostname to userhost')
```

Comment:
I don't find this log message useful by itself.  I'd either delete it, or add the host.

Reply:
Sorry, it was a debug message.